### ISSUE TITLE
WARNING: NVIDIA Unable to query number of CUDA devices!

### ISSUE THREAD

**Issue by Libbzter** [fireice-uk/xmr-stak#1081]
Please provide as much as possible information to reproduce the issue.

# Basic information
  - Intel 15
  - Geforce 860M

# Compile issues
  - Win 8.1
  ```
# This is what I get when I try to run. It does not create a Nvidia.txt file. 
I have tried updating and rolling back drivers for video card. Same. 
I feel like i`m missing something simple. 


-------------------------------------------------------------------
xmr-stak 2.2.0 c4400d19

Brought to you by fireice_uk and psychocrypt under GPLv3.
Based on CPU mining code by wolf9466 (heavily optimized by fireice_uk).
Based on NVIDIA mining code by KlausT and psychocrypt.
Based on OpenCL mining code by wolf9466.

Configurable dev donation level is set to 2.0%

You can use following keys to display reports:
'h' - hashrate
'r' - results
'c' - connection
-------------------------------------------------------------------
[2018-02-14 23:20:03] : Start mining: MONERO
WARNING: NVIDIA Unable to query number of CUDA devices!
[2018-02-14 23:20:47] : WARNING: backend NVIDIA disabled.
[2018-02-14 23:20:47] : WARNING: No AMD OpenCL platform found. Possible driver i
ssues or wrong vendor driver.
[2018-02-14 23:20:47] : WARNING: backend AMD disabled.
[2018-02-14 23:20:47] : Starting 1x thread, affinity: 0.
[2018-02-14 23:20:47] : hwloc: memory pinned
[2018-02-14 23:20:47] : Starting 1x thread, affinity: 2.
[2018-02-14 23:20:47] : hwloc: memory pinned
[2018-02-14 23:20:47] : Fast-connecting to pool.bytecoin.party:5555 pool ...
[2018-02-14 23:20:47] : Pool pool.bytecoin.party:5555 connected. Logging in...
[2018-02-14 23:20:48] : Difficulty changed. Now: 2000.
[2018-02-14 23:20:48] : Pool logged in.
[2018-02-14 23:21:05] : Result accepted by the pool.

**Comment by psychocrypt** [comment]
please install the cuda driver from https://developer.nvidia.com/cuda-downloads

**Comment by Libbzter** [comment]
I installed the Cuda driver ( I think). Still getting the same output

**Comment by Libbzter** [comment]
Hi, thanks for the response. After installing the cuda driver, I am still getting the same thing.

**Comment by marimarluna** [comment]
I have installed the cuda 9.1 and still does not work I have that same error

**Comment by Libbzter** [comment]
Is there a solution to this? I am getting pretty frustrated, trying to figure this out.

**Comment by JerichoJones** [comment]
DDU, re-install NVIDIA drivers. CUDA should not be needed on Windows. That being said, none of us run v8.1.

If that fails, try [REDDIT/r/moneromining](https://www.reddit.com/r/MoneroMining/new/)

**Comment by Libbzter** [comment]
@JerichoJones What will DDU do that my numerous installs, re-installs, rollbacks, have done? CUDA is not needed? I don`t understand why psychocrypt would be telling me that I needed those drivers. I`m confused. What gives?

**Comment by Libbzter** [comment]
@JerichoJones Well, that did`nt work either. Is there another GPU mining software that you would suggest trying?

**Comment by Libbzter** [comment]
I`m pretty sure that there is a problem with my video card. Thanks for your time.

**Comment by Xaac** [comment]
same issue for me with v1.22 fresh windows install and latest drivers.
however when i try running v1.08 it works, so problem is not with my drivers and ddu probably is not going to fix the problem. it is not an easy task to ddu 12 gpu rig, you ask as if it is a simple task, it takes half my day ddu a rig. p

**Comment by Spudz76** [comment]
v1.22 and v1.08 are not xmr-stak versions?

CUDA comes with drivers.  If you need CUDA 8.0 (*Fermi you do) then install the last available CUDA8 driver, 386.28

You also need the SDK if compiling, but not for running a release binary.  For the release binary you must run the correct driver, 386.28, with CUDA8 bundled, the end.
